src/HybridDb.Tests/Migrations/DocumentMigrationRunnerTests.cs

```diff
@@ -429,6 +430,49 @@ public async Task MigratesSelectedIdsAndIdPrefixCombined()
             migratedIds.ShouldBeLikeUnordered("aatest");
         }
 
+        [Fact]
+        public async Task MigratesWithNull()

```

Comment:
Better name for test. Mayby, `DeletesDocumentIfMigrationReturnsNull` or something similar